alchemy-web/src/content/docs/providers/cloudflare/worker.mdx

```diff
@@ -5,80 +5,179 @@ description: Learn how to deploy, configure, and manage Cloudflare Workers using
+  cwd: "./apps/my-api",                   // Project root directory (defaults to process.cwd())
+  entrypoint: "./src/api.ts",             // Main entrypoint for bundling
+  format: "esm",                          // Module format: 'esm' (default) or 'cjs'
+  // compatibilityDate defaults to the SDK's pinned date; override only when you need a specific runtime
+  compatibilityDate: "2025-09-13",        // Workers runtime version (uses DEFAULT_COMPATIBILITY_DATE when omitted)
+  compatibilityFlags: ["nodejs_compat"],  // Low-level runtime flags; consider using `compatibility` preset instead
+  compatibility: "node",                  // Compatibility preset (optional) — expands flags for common use-cases like Node.js compatibility
+  adopt: true,                            // Adopt existing Worker if present; falls back to scope.adopt when undefined
   bindings: {
-    CACHE: kv,
-    USERS: users,
+    CACHE: cache,                         // KV Namespace binding
+    STORAGE: storage,                     // R2 bucket binding
+    API_KEY: alchemy.secret("secret-value") // Plain-text env binding (converted to plain_text)
+    // Other binding types include: DurableObjectNamespace, Queue, Secret, Assets, Container, Workflow
+  },
+  observability: {
+    enabled: true                         // Enable worker logs / observability (default: true)
+  },
+  // Upload source maps to improve stack traces (set to false to disable)
+  sourceMap: true,
+  // Optional runtime limits — increase only when necessary after monitoring
+  limits: {
+    cpu_ms: 50_000                        // Max CPU time in ms (default: 30_000)
   },
+  // Other Advanced 🧪 & commonly used options
+  placement: { mode: "smart" },          // Smart placement to optimize network placement for latency
+  assets: { path: "./public", run_worker_first: false }, // Static assets configuration
+  crons: ["0 0 * * *"],                  // Scheduled cron triggers (standard cron syntax)
+  eventSources: [                        // Background event sources (queues, streams)
+    { queue: taskQueue, settings: { batchSize: 15, maxConcurrency: 3 } }
+  ],
+  routes: ["api.example.com/*"],          // Route patterns to bind the worker
+  domains: ["example.com"],               // Custom domains to bind (can be objects with zoneId/adopt)
+  version: "pr-123",                      // Publish as preview version with label
+  dev: { port: 8787 },                    // Local development options (port, remote, tunnel)
+  tunnel: true,                           // Expose the worker via a Cloudflare Tunnel.
+  namespace: await DispatchNamespace("my-dispatch-ns", {
+    namespace: "production-dispatch"
+  }),                    // Deploy to dispatch namespace (string | DispatchNamespace)
+  noBundle: false,                        // Disable bundling (deploy files as-is)
+  rules: [{ globs: ["**/*.wasm"] }],      // Additional bundle rules
```

Comment:
I don't think we support this do we?

Reply:
This is in the `worker.ts` types 
```
 /**
   * Rules for adding additional files to the bundle.
   *
   * If {@link noBundle} is false | undefined, this will be ignored.
   *
   * @default - all .js, .mjs, and .wasm files under the entrypoint directory
   */
  rules?: {
    globs: string[];
  }[];
  ``` 
  
  And it looks like its passed into the normalize `normalizeWorkerBundle` function